packages/format-message-formats/types.d.ts

```diff
@@ -0,0 +1,18 @@
+interface Format {
+    [option: string]: any
+}
+
+interface FormatGroup {
+    default: Format,
+
+    [style: string]: Format
+}
+
+export declare const number: FormatGroup;
+export declare const date: FormatGroup;
+export declare const time: FormatGroup;
+export declare const duration: FormatGroup;
```

Comment:
This is probably fine for now. It would be more specific though to say number's formats are `Intl.NumberFormat` options rather than `{ [string]: any }`. I'm not sure TypeScript has an easy way to reference those if defined anyway.

Reply:
Actually it has, I just changed it.